config/initializers/omniauth.rb

```diff
@@ -1,15 +1,24 @@
 OmniAuth.config.logger = Rails.logger
 
+# Options for google oauth
+Orientation::Application.config.x.oauth_options = {
```

Comment:
We already have a `config.orientation` hash built from orientation.yml, let's move these configuration options there rather than create yet another configuration object.

I'd like to centralize things over there as much as possible.